### ISSUE TITLE
Feature Request: Simplify user setup

### ISSUE THREAD

**Issue by LeLunZ** [MrAsterisco/homebridge-hisense-tv#65]
**Is your feature request related to a problem? Please describe:**
Often when a user configures the plugin it doesn't show any Inputs because the TV wasn't online yet, and the plugin couldn't grab the state. 
You then need to turn on the TV and refresh the home app 1 or 2 times.

**Describe the solution you'd like:**
Two possible solutions:

- In the initialization process of the plugin we could turn on the TV, grab all the states we need and then automatically shut it down. Only after that we publish the Accessory as external plugin.

- Or instead of having the current setup where the inputs get fetched from the tv, a user would have to configure all the inputs in the config.

**Comment by LeLunZ** [comment]
I will probably try out the first solution, I think this could work really well.

**Comment by MrAsterisco** [comment]
I agree, @LeLunZ ! But perhaps, let's make sure we don't _blast_ people ears while doing so: I can imagine that if, for any reason, your Homebridge instance restarts at 2am, you don't want your TV to turn on and start playing whatever late TV show is airing at that time 😃 

Perhaps we can turn it on and immediately set the volume to 0?

**Comment by LeLunZ** [comment]
So, after a long time a progress update on this. In V4, I handled this differently than previously described here.

---

We now cache the input sources on disk and can just republish the accessory instantly even if the tv is turned off.

The first time when no cache is available, we wait with publishing the accessory till the tv is on. But I have added enough warning logs that a user should get, and in the setup guide I also added a step where it clearly says the tv needs to be turned on.


This now makes input available instantly after the first time setting up the plugin.
